### PR TITLE
[updated] pda changes

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

You can get a nonmodular one in loadout if you want
This should make rnd and other venders or disk more usefull as they now can sell modular tablets and stuff

## Changelog
:cl:
/:cl:

### CODE REVIEW

**Comment by Kazkin** [comment]
I still remain stubbornly unconvinced of using the old system over the much sleeker and better designed new system.

**Comment by Trilbyspaceclone** [comment]
Its no longer replacing anything. check the code!

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -4,8 +4,9 @@
 var/global/list/obj/item/device/pda/PDAs = list()
 
 /obj/item/device/pda
-	name = "\improper PDA"
-	desc = "A portable microcomputer by Thinktronic Systems, LTD. Functionality determined by a preprogrammed ROM cartridge."
+	name = "\improper outdated PDA"
+	desc = "A portable microcomputer by NanoTrason, do to many people finding it hard to repair and replace phased it out to the now used modula PDAs made by Soteria. \
+	Functionality determined by a preprogrammed ROM cartridge rather then any software download making it a marketing nightmare."
```
Comment (r4d6) [inline code/game/objects/items/devices/PDA/PDA.dm:9]: > A portable microcomputer by NanoTrason, **due** to many people finding it hard to repair and replace **, it was phased out in favor of the more modular models** made by Soteria. 
> Functionality **is** determined by a preprogrammed ROM cartridge rather then any software download **,** making it a marketing nightmare.

Just a better description. the changes are in **bold** and include commas

**Review by r4d6 — COMMENTED** [review]
Just some corrections to the description.

Also you probably should change the description of the PR and the title. Because the way you are wording it mean that everyone start with non-modular PDA with modular PDAs being available in the loadout screen. I was about to ask that you add a customization option to the PDA, before I realised the PR was currently doing the exact opposite to what it said : Adding non-modular PDAs to the loadout.

**Pull request closed without merging**